conf/app.routes

```diff
@@ -1,12 +1,12 @@
 # microservice specific routes
 
-POST                    /movements/arrivals                 controllers.MovementsController.post
-GET                     /movements/arrivals                 controllers.MovementsController.getArrivals()
+POST                    /movements/arrivals                                     controllers.MovementsController.post
+GET                     /movements/arrivals                                     controllers.MovementsController.getArrivals()
 
-GET                    /movements/arrivals/:arrivalId                 controllers.MovementsController.getArrival(arrivalId: ArrivalId)
-PUT                    /movements/arrivals/:arrivalId                 controllers.MovementsController.putArrival(arrivalId: ArrivalId)
+GET                     /movements/arrivals/:arrivalId                          controllers.MovementsController.getArrival(arrivalId: ArrivalId)
+PUT                     /movements/arrivals/:arrivalId                          controllers.MovementsController.putArrival(arrivalId: ArrivalId)
 
-GET                    /movements/arrivals/:arrivalId/messages/:messageId           controllers.MovementsController.getMessage(arrivalId: ArrivalId, messageId: MessageId)
+GET                     /movements/arrivals/:arrivalId/messages/:messageId      controllers.MovementsController.getMessage(arrivalId: ArrivalId, messageId: MessageId)
 
-POST                    /movements/arrivals/:arrivalId/messages                   controllers.MessagesController.post(arrivalId: ArrivalId)
-POST                    /movements/arrivals/:messageSender/goods-released    controllers.GoodsReleasedController.post(messageSender: MessageSender)
+POST                    /movements/arrivals/:messageSender/messages/eis         controllers.MessageResponseController.post(messageSender: MessageSender)
```

Comment:
Like that we're now sticking eis on the end but would prefer this in an eis.routes file that's imported (clear this isn't publicly accessible) Something to keep in mind for future